port/aix/omrosdump_helpers.c

```diff
@@ -132,6 +132,9 @@ genSystemCoreUsingGencore(struct OMRPortLibrary *portLibrary, char *filename)
 	BOOLEAN filenameWasEmpty = FALSE;
 	struct coredumpinfo coreDumpInfo;
 	int rc = -1;
+#if defined(DUMP_DBG)
+	int saved_errno;
```

Comment:
camel case recommended as per coding standards; also, to be consistent with the surrounding code. https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#variables

```suggestion
	int savedErrno;
```